pkg/monitor/monitor/util.go

```diff
@@ -44,6 +44,14 @@ const (
 	defaultReplicaExternalLabelName = "prometheus_replica"
 )
 
+func GetTlsAssetsSecretName(name string) string {
+	return fmt.Sprintf("%s-tls-assets", prefixedName(name))
+}
+
+func prefixedName(name string) string {
```

Comment:
Where is this used?

Reply:
GetTlsAssetsSecretName method and prefixedName can be  merged.